### ISSUE TITLE
Calculate checksum

### ISSUE THREAD

**Issue by Hogfeldt** [Hogfeldt/bioconda_recipe_gen#51]
The checksum arguments should be optional and we should calculate the checksum ourself.

**Comment by Hogfeldt** [comment]
A valid md5 checksum can be calculated with the following code:
```
>>> import hashlib
>>> hashlib.md5(open("bsdiff4-1.1.4.tar.gz", 'rb').read()).hexdigest()
'29f6089290505fc1a852e176bd276c43'
```

**Comment by Hogfeldt** [comment]
I think it could be nice if we just calculate a checksum and adds it, when we add the source url in the Recipe class. 

The function which calculates the checksum could be placed in `utils.py`. We should decide which hashing function we would like to use.

If we always calculate a checksum the arguments `--sha` and `--md5` is redundant and could probably just be removed.

**Comment by Hogfeldt** [comment]
The feature is implemented in the commit d4cb96f38e0e9 and just needs to be tested before a pull request can be made.
